terraform/provider/pkg/provider/data_resolved_app_configs_test.go

```diff
@@ -37,7 +38,30 @@ func exportRsaPrivateKeyAsPemStr(privkey *rsa.PrivateKey) string {
 	return string(privkey_pem)
 }
 
+func stashEnv() []string {
```

Comment:
saw this in a lot of provider code online and thought it was a good idea and easier to test and cleanup how environment variables were affecting the providers configuration